ios/PresentationLayer/Recipes/Sources/Recipes/RecipesFlowController.swift

```diff
@@ -11,15 +11,34 @@ enum RecipesFlow: Flow, Equatable {
     case recipes(Recipes)
     
     enum Recipes: Equatable {
+        static func == (lhs: RecipesFlow.Recipes, rhs: RecipesFlow.Recipes) -> Bool {
+            switch (lhs, rhs) {
+            case (.showCounter, .showCounter): true
+            case (.showBooks, .showBooks): true
+            case (.showSkeleton, .showSkeleton): true
+            case (.showImages, .showImages): true
+            case (.showMaps, .showMaps): true
+            case (.showMedia, .showMedia): true
+            case (.presentPickerModal, .presentPickerModal): true
+            default: false
+            }
+        }
+        
         case showCounter
         case showBooks
 //        case showRocketLaunches
         case showSkeleton
         case showImages
         case showMaps
+        case showMedia
+        case presentPickerModal(delegate: MediaPickerSource)
     }
 }
 
+public protocol MediaPickerSource: AnyObject {
+    var media: Binding<[MediaType]> { get }
+}
```

Comment:
This is now defined both here and in `MediaPickerView` 🤔 Can it be removed here?